src/applications/appeals/995/containers/App.jsx

```diff
@@ -83,7 +84,11 @@ export const App = ({
             ...formData,
             benefitType: subTaskBenefitType,
           });
-        } else if (loggedIn && formData.benefitType) {
+        } else if (
+          loggedIn &&
+          (!isOutsideForm(pathname) || formData.internalTesting) &&
```

Comment:
might be nice to leave a comment here explaining the `internalTesting` stuff so that we (i) can remember what was happening if we revisit this in the future 😬

Reply:
Done! Please let me know if the comment is clear enough

Reply:
works for me! thank you!